[17:22:24] [DEBUG] Running client sanity checks...
[17:22:25] [DEBUG] No setup function found in play
[17:22:25] [DEBUG] Found extension class Music in play: Attempting to load
[17:22:25] [DEBUG] Added callback: Music._menu
[17:22:25] [DEBUG] Added callback: Music._pause
[17:22:25] [DEBUG] Added callback: Music._resume
[17:22:25] [DEBUG] Added callback: Music._skip
[17:22:25] [DEBUG] Added callback: Music._stop
[17:22:25] [DEBUG] Added callback: Music.play
[17:22:25] [DEBUG] Loaded Extension: play
[17:22:26] [DEBUG] No setup function found in askgpt
[17:22:26] [DEBUG] Found extension class Gpt in askgpt: Attempting to load
[17:22:26] [DEBUG] Added callback: Gpt._askgpt
[17:22:26] [DEBUG] Added callback: Gpt._img
[17:22:26] [DEBUG] Loaded Extension: askgpt
[17:22:26] [DEBUG] No setup function found in askbard
[17:22:26] [DEBUG] Found extension class Bard in askbard: Attempting to load
[17:22:26] [DEBUG] Added callback: Bard._askbard
[17:22:26] [DEBUG] Added callback: Bard._endbard
[17:22:26] [DEBUG] Loaded Extension: askbard
[17:22:26] [DEBUG] No setup function found in noi_chu
[17:22:26] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[17:22:26] [DEBUG] Added callback: NoiChu._reset
[17:22:26] [DEBUG] Added callback: NoiChu._world_setup
[17:22:26] [DEBUG] Loaded Extension: noi_chu
[17:22:26] [DEBUG] No setup function found in db_refesh
[17:22:26] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[17:22:26] [DEBUG] Added callback: Database.dbv1_command
[17:22:26] [DEBUG] Added callback: Database.dbv2_command
[17:22:26] [DEBUG] Loaded Extension: db_refesh
[17:22:26] [DEBUG] Added callback: __join
[17:22:26] [DEBUG] Added callback: __leave
[17:22:26] [DEBUG] Added callback: _about
[17:22:26] [DEBUG] Added callback: _help
[17:22:26] [DEBUG] Added callback: _refresh
[17:22:26] [DEBUG] Added callback: _setup
[17:22:26] [DEBUG] Added callback: _starup
[17:22:26] [DEBUG] 7 callbacks have been loaded from __main__.
[17:22:26] [DEBUG] Added callback: _disconnect
[17:22:26] [DEBUG] Added callback: _on_websocket_ready
[17:22:26] [DEBUG] Added callback: on_autocomplete_completion
[17:22:26] [DEBUG] Added callback: on_autocomplete_error
[17:22:26] [DEBUG] Added callback: on_command_completion
[17:22:26] [DEBUG] Added callback: on_command_error
[17:22:26] [DEBUG] Added callback: on_component_completion
[17:22:26] [DEBUG] Added callback: on_component_error
[17:22:26] [DEBUG] Added callback: on_error
[17:22:26] [DEBUG] Added callback: on_modal_completion
[17:22:26] [DEBUG] Added callback: on_modal_error
[17:22:26] [DEBUG] Added callback: on_resume
[17:22:26] [DEBUG] 12 callbacks have been loaded from Client.
[17:22:26] [DEBUG] Attempting to login
[17:22:27] [DEBUG] Caching ingested rate limit data for: f7ead6a7674e5a323d93786263b66cb1
[17:22:27] [DEBUG] GET /users/@me Received 200 :: [49/50 calls remaining]
[17:22:27] [DEBUG] Caching ingested rate limit data for: f7ead6a7674e5a323d93786263b66cb1
[17:22:27] [DEBUG] GET /oauth2/applications/@me Received 200 :: [49/50 calls remaining]
[17:22:27] [DEBUG] Caching ingested rate limit data for: f7ead6a7674e5a323d93786263b66cb1
[17:22:27] [DEBUG] GET /gateway Received 200 :: [49/50 calls remaining]
[17:22:27] [INFO] Shard ID 0 | Starting Shard
[17:22:27] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[17:22:28] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[17:22:28] [INFO] Shard ID 0 | Gateway connection established
[17:22:28] [DEBUG] Shard ID 0 | Session ID: b936fb3fafbfba6f1c968a7b48a9b840 Trace: ['["gateway-prd-us-east1-d-h20c",{"micros":127075,"calls":["id_created",{"micros":800,"calls":[]},"session_lookup_time",{"micros":16300,"calls":[]},"session_lookup_finished",{"micros":13,"calls":[]},"discord-sessions-prd-2-291",{"micros":108683,"calls":["start_session",{"micros":61459,"calls":["discord-api-866cdcc96c-46dxk",{"micros":56442,"calls":["get_user",{"micros":7353},"get_guilds",{"micros":7975},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":220},"authorized_ip_coro",{"micros":10}]}]},"starting_guild_connect",{"micros":58,"calls":[]},"presence_started",{"micros":334,"calls":[]},"guilds_started",{"micros":198,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":46597,"calls":[]},"connect_finished",{"micros":46603,"calls":[]},"build_ready",{"micros":26,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":1,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}]']
[17:22:28] [DEBUG] Dispatching Event: websocket_ready
[17:22:29] [DEBUG] Caching ingested rate limit data for: f7ead6a7674e5a323d93786263b66cb1
[17:22:29] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [49/50 calls remaining]
[17:22:29] [DEBUG] Caching ingested rate limit data for: f7ead6a7674e5a323d93786263b66cb1
[17:22:29] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [49/50 calls remaining]
[17:22:29] [DEBUG] 0 is already up-to-date with 18 commands.
[17:22:29] [DEBUG] Sync of 1 scopes took 0.6527280000009341 seconds
[17:22:29] [DEBUG] Dispatching Event: startup
[17:22:29] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1700562149343, "activities": [{"name": "l\u1ec7nh /help \u0111\u1ec3 gi\u00fap \u0111\u1ee1", "type": 0}], "status": "idle", "afk": false}}
[17:22:30] [DEBUG] 1174695146214928500::play should be called
[17:22:31] [DEBUG] Dispatching Event: message_create
[17:22:31] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[17:22:31] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE3NjQ2NzY3MjIwMDkxMjkwNzpJTVRPU21wY0JzdXlvTDFVclZ0MFNuQkdwUkpZMUFmdERtNmpqSHBiU004WnlmMXg3M1hjbExjamJrTWVSOUZNYXRWRnRPMGVKeFJMU2N4TWFHdkV5MkdQakg5bkVxOERHcHVZQ29DNW53ZlRReFpqdmRYS05CTmswcG83UGxuQw/callback Received 204 :: [4/5 calls remaining]
[17:22:31] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1176466591056142377, "self_mute": false, "self_deaf": false}}
[17:22:31] [DEBUG] Waiting for voice connection data...
[17:22:31] [DEBUG] Dispatching Event: voice_user_join
[17:22:31] [DEBUG] Attempting to initialise voice gateway...
[17:22:31] [DEBUG] Voice Connection has identified itself to Voice Gateway
[17:22:32] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[17:22:32] [DEBUG] IP Discovery in progress...
[17:22:32] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x01\xb6\x0e116.100.185.240\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xd5\xd6'
[17:22:32] [DEBUG] IP Discovered: 116.100.185.240 #54742
[17:22:32] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.185.240", "port": 54742, "mode": "xsalsa20_poly1305_lite"}}}
[17:22:32] [DEBUG] Starting UDP Keep Alive
[17:22:32] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[17:22:32] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[17:22:32] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[17:22:35] [DEBUG] Dispatching Event: command_error
[17:22:35] [DEBUG] Dispatching Event: command_completion
[17:22:35] [DEBUG] Dispatching Event: error
[17:22:35] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/watch?v=us8kABcQ_Pk'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/watch?v=us8kABcQ_Pk'}
[17:22:35] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 199, in play
    embed.set_author('üìÄ ƒêang Ch∆°i Nh·∫°c')
    ^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'set_author'

[17:22:35] [DEBUG] Sending heartbeat every 13.75 seconds
[17:22:35] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.7470217813569149}
[17:22:35] [DEBUG] ‚ù§ Voice Connection is sending Heartbeat
[17:22:35] [DEBUG] ‚ù§ Heartbeat acknowledged after 0.08131 seconds
[17:22:42] [DEBUG] Sending heartbeat every 41.25 seconds
[17:22:42] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[17:22:42] [DEBUG] Shard ID 0 | ‚ù§ Gateway is sending a Heartbeat
[17:22:43] [DEBUG] Shard ID 0 | ‚ù§ Received heartbeat acknowledgement from gateway
[17:22:49] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.4692505930976889}
[17:22:49] [DEBUG] ‚ù§ Voice Connection is sending Heartbeat
[17:22:49] [DEBUG] ‚ù§ Heartbeat acknowledged after 0.07885 seconds
[17:22:59] [DEBUG] Dispatching Event: disconnect
[17:22:59] [DEBUG] Stopping the bot.
[17:22:59] [INFO] Shard ID 0 | Stopping Shard
